src/components/ExportKeysForm.vue

```diff
@@ -156,40 +157,38 @@ export default {
     QrcodeCapture,
     QrcodeScannerDialog
   },
-  data () {
-    return {
-      passphrase: '',
-      showQrcodeScanner: false,
-      keys: []
-    }
-  },
-  computed: {
-    className: () => 'export-keys-form'
-  },
-  methods: {
-    onDetectQrcode (passphrase) {
-      this.onScanQrcode(passphrase)
-    },
 
-    onDetectQrcodeError (error) {
+  setup () {
+    const passphrase = ref('')
+    const showQrcodeScanner = ref(false)
+    const store = useStore()
+    const { t } = useI18n()
+    const keys = ref([])
+
+    const className = computed(() => {
+      return 'export-keys-form'
+    })
+
+    function onDetectQrcode (passphrase) {
```

Comment:
Functions inside `setup()` should be declared by using `const`. Same for other functions below.

```suggestion
    const onDetectQrcode = (passphrase) => {
```

Reply:
Not resolved. Use `const`